spec/acceptance/zabbix_host_spec.rb

```diff
@@ -132,8 +132,36 @@ class { 'zabbix':
         end
       end
 
+      it_behaves_like 'an idempotent resource' do
+        let(:manifest) do
+          <<-EOS
+          zabbix_host { 'test5.example.com':
+            ipaddress   => '127.0.0.5',
+            use_ip      => false,
+            port        => 1051,
+            groups      => ['Virtual machines'],
+            templates   => #{template},
+            macros      => [],
```

Comment:
Please drop these as well, they don't add any value, just more useless API calls

Reply:
Maybe I was wrong? Revert that and it's good to go. Sorry 😄